config/default/editor.editor.filtered_html.yml

```diff
@@ -116,7 +118,7 @@ settings:
         - '<div class>'
         - '<blockquote cite class>'
         - '<ul type class>'
-        - '<ol start type reversed class>'
```

Comment:
Don't we want to keep these @joewhitsitt ?

Reply:
It causes import errors there. I previously tried to commit it back in. Just tried moving those to here and didn't have an issue with import and i think it preserved it.

Reply:
It's working, thanks!